### PR TITLE
Clarify slurmdbd steps

### PR DESCRIPTION
**Pull request by tahliar**
### Description

Updated Slurm database steps after testing.

### Are there any relevant issues/feature requests?

* bsc#1173150
* jsc#DOCTEAM-1343

### Which product versions do the changes apply to?

- [x] SLE HPC 15 next *(current `main`, no backport necessary)*
- [x] SLE HPC 15 SP6
- [x] SLE HPC 15 SP5
- [x] SLE HPC 15 SP4
- [x] SLE HPC 15 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by e4t** [comment]
To me, this looks fine.  @mslacken, what do you think ?

**Review by mslacken — APPROVED** [review]
looks good

File: xml/slurm.xml [hunk 1]
```diff
@@ -363,26 +376,32 @@ AccountingStorageHost=localhost</screen>
...
+       is running, <literal>slurmctld</literal> fails because it cannot connect to the database.
+      </para>
     </step>
     <step>
      <para>
       Restart <literal>slurmctld</literal>:
      </para>
 <screen>&prompt.mgmt;systemctl restart slurmctld</screen>
+     <para>
+      This creates the &slurm; database, and adds the cluster to the database (using the
```
Comment (dariavladykina) [inline xml/slurm.xml:394]:
```suggestion
      This creates the &slurm; database and adds the cluster to the database (using the
```

**Review by dariavladykina — COMMENTED** [review]
LGTM, just a minor nit. Thanks!

**Pull request merged**